src/components/VideoChatButtonAndMenu/BaseVideoChatButtonAndMenu.js

```diff
@@ -53,7 +43,7 @@ class VideoChatButtonAndMenu extends Component {
                 text: props.translate('videoChatButtonAndMenu.googleMeet'),
                 onPress: () => {
                     this.toggleVideoChatMenu();
-                    Linking.openURL(CONST.NEW_GOOGLE_MEET_MEETING_URL);
+                    Linking.openURL(this.props.newGoogleMeetingUrl || CONST.NEW_GOOGLE_MEET_MEETING_URL);
```

Comment:
Umm, let's pass `CONST.NEW_GOOGLE_MEET_MEETING_URL` from index.js
I think it improves code readability.